glide.lock

```diff
@@ -61,6 +63,8 @@ imports:
   - i18n/bundle
   - i18n/language
   - i18n/translation
+- name: github.com/nu7hatch/gouuid
```

Comment:
What's this for? I see it being used in the enterprise PR but is there a reason we can't use the guid generator we already have?